test/TestHelpers.jl

```diff
@@ -83,15 +83,13 @@ function challenge_prompt(cmd::Cmd, challenges; timeout::Integer=10, debug::Bool
     with_fake_pty() do slave, master
         p = spawn(detach(cmd), slave, slave, slave)
         # Kill the process if it takes too long. Typically occurs when process is waiting for input
-        @async begin
+        t = @async begin
             sleep(timeout)
             kill(p)
             close(master)
         end
         try
             for (challenge, response) in challenges
-                process_exited(p) && error("Too few prompts. $(format_output(out))")
-
```

Comment:
I couldn't manage to trigger this error. Most of the time this loop is blocked by `readuntil` which will abend when the process is aborted and trigger the "Could not locate challenge" error.